x/sync/README.md

```diff
@@ -36,7 +36,7 @@ This message is sent from the server to the client in response to a `SyncGetChan
 
 ## Algorithm
 
-For each proof it receives, the sync client tracks the root hash of the revision associated with the proof's key-value pairs. For example, it will store information that says something like, "I have all of the key-value pairs that are in range [`start`, `end`] at the revision with root `root_hash`" for some keys `start` and `end`. Note that `root_hash` is the root hash of the revision that the client is trying to sync to, not the root hash of its own (incomplete) database. Tracking the revision associated with each downloaded key range, as well as using data in its own (incomplete) database, allows the client to figure out which key ranges are not up to date and need to be synced. The hash of the incomplete database on a client is never sent anywhere because it does not represent a root hash of any revision.
+For each proof it receives, the sync client tracks the root hash of the revision associated with the proof's key-value pairs. For example, it will store information that says something like, "I have all of the key-value pairs that are in range [`start`, `end`] for the revision with root `root_hash`" for some keys `start` and `end`. Note that `root_hash` is the root hash of the revision that the client is trying to sync to, not the root hash of its own (incomplete) database. Tracking the revision associated with each downloaded key range, as well as using data in its own (incomplete) database, allows the client to figure out which key ranges are not up to date and need to be synced. The hash of the incomplete database on a client is never sent anywhere because it does not represent a root hash of any revision.
```

Comment:
would it be possible to do a hard wrap of like 80 or 100 chars? I find this nice for reviewing text in markdown.

Reply:
I use this in my `vscode` settings for this kinda stuff - but don't treat it as a hard rule
```json
"editor.rulers": [
    80
],
```

Reply:
added newlines